### PR TITLE
Made hydrogen peroxide to work like disinfectant

### PR DESCRIPTION
**Pull request by RarkGrames**
```SUMMARY: Bugfixes "Made hydrogen peroxide to work like disinfectant"```

#### Purpose of change
Upon using a small bottle of hydrogen peroxide found in the game, it uses up all the 0,25 liter for one wound, instead working like disinfectant.

#### Describe the solution
- Changed it to have 10 charges, just like the disinfectant bottles
- Changed the HP unit amount tenfold in recipes that use it as a component
- Made the crafting recipe from concentrated HP use up less water than the resulting mixture

### CODE REVIEW

**Comment by RarkGrames** [comment]
> 
> 
> Weight should be updated - it is now 2.76 kg.

Changed it.

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -253,9 +253,10 @@
     "name": "hydrogen peroxide",
     "name_plural": "hydrogen peroxide",
     "description": "Dilute hydrogen peroxide, for use as a disinfectant and for bleaching hair or textiles.  Foams a little when in contact with organic matter, but otherwise harmless.",
-    "weight": 275,
+    "weight": 27.5,
```
Comment (Reclusive-reptile) [inline data/json/items/comestibles/med.json]: weight must be an integer, no decimals.

Comment (RarkGrames) [reply data/json/items/comestibles/med.json]: Changed it.

**Comment by paulenka-aleh** [comment]
Just to double check... Currently 'just' disinfectant increases healing rate. Hydrogen peroxide in fact disrupts tissue healing. There are a lot of references, the first one given by google: https://articles.mercola.com/sites/articles/archive/2016/06/11/amp/hydrogen-peroxide-wound-cleaning.aspx

So hydrogen peroxide shouldn't be doing it, it is good only for cleaning.

**Comment by JosephCarrington** [comment]
> Just to double check... Currently 'just' disinfectant increases healing rate. Hydrogen peroxide in fact disrupts tissue healing. There are a lot of references, the first one given by google: https://articles.mercola.com/sites/articles/archive/2016/06/11/amp/hydrogen-peroxide-wound-cleaning.aspx
> 
> So hydrogen peroxide shouldn't be doing it, it is good only for cleaning.

That article is from a 'natural health' website. While it may be correct that hydrogen peroxide is not ideal for cleaning wounds, it's hard to take it very seriously.

**Comment by RarkGrames** [comment]
> 
> 
> Just to double check... Currently 'just' disinfectant increases healing rate. Hydrogen peroxide in fact disrupts tissue healing. There are a lot of references, the first one given by google: https://articles.mercola.com/sites/articles/archive/2016/06/11/amp/hydrogen-peroxide-wound-cleaning.aspx
> 
> So hydrogen peroxide shouldn't be doing it, it is good only for cleaning.

As far as I know disinfectants themselves don't accelerate healing rate of a wound, they prevent slow healing caused by germs and contaminants, so the wound don't get festered. Hydrogen peroxide might damage some tissue, but this minimal tissue damage would be negligible compared to a festering wound. So it might cause slower healing compared to rubbing alcohol or iodine tincture, but from a game mechanics perspective I don't think that it is needed to differentiate it from other kind of disinfectants.

**Comment by LordSunder** [comment]
Slightly outside the scope of the pr, but another potent, naturally occurring disinfectant is honey, which contains moderate quantities of hydrogen peroxide from bee saliva. See https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3504486/ for details. The sugar in the honey might also help in fighting blob-borne infections like zombie bites by drawing vital water out of the blob (and your surrounding tissues, unfortunately).
Only problem is that honey is a comestible, so unless you swab it on a rag or something, it'd need a new menu at least, but if hydrogen peroxide works, forest honey should at least work as a weak disinfectant.

**Comment by RarkGrames** [comment]
@LordSunder You are right, but this PR is a bugfix about the bottled HP found in the game. You can open a separate issue with the sources you found, or in case you know how to work in json (it's really easy), you can add the recipe and other stuff.

**Pull request merged**